Source/WebKit/GPUProcess/GPUProcess.cpp

```diff
@@ -91,6 +91,9 @@ GPUProcess::GPUProcess()
     : m_idleExitTimer(*this, &GPUProcess::tryExitIfUnused)
 {
     RELEASE_LOG(Process, "%p - GPUProcess::GPUProcess:", this);
+#if ASSERT_ENABLED && PLATFORM(COCOA)
+    CoreAudioSharedUnit::singleton().allowStarting();
```

Comment:
I am confused how the assertion is not hit for WK1 since the code resides in WebCore and we only call `allowStarting()` from WK2. Isn't BaseAudioSharedUnit called when using WK1 as well. Also, do we still have a flag to disable media in the GPUProcess?

Reply:
We do not support capture in WK1.
We have a flag to move back capture to UIProcess but it is no longer enabled anywhere AFAIK.
Plan is to remove that flag to reduce UIProcess IPC exposure.